### PR TITLE
Ap 2618/one installation package for both 32bit and 64 bit

### PR DESCRIPTION
**Pull request by junos**
## Tracker Ticket

* https://quandl.atlassian.net/browse/AP-2618

## Description

* Investigate combining 32 and 64bit Excel installers

## Note to the reviewer

* None

## Note to QA

* None

## Dependency

* None

### CODE REVIEW

File: Quandl.Excel.Console/SetupHelp.cs [hunk 1]
```diff
@@ -11,17 +14,49 @@ public class SetupHelp
...
 
         public static string ValueKeyName { get; } = "OPEN";
 
         public static void RegisterExcelAddin()
         {
             foreach (var subKey in AddinRegisterKeys())
             {
-                RemoveAvailableOpenOption(subKey); // Remove any existing old keys
-                SetAvailableOpenOption(subKey); // Add in any new keys
+                _version = subKey["key"].ToString();
```
Comment (mbasset) [inline Quandl.Excel.Console/SetupHelp.cs]: what is this doing?, should it be passed into the methods below?

File: Quandl.Excel.Console/SetupHelp.cs [hunk 2]
```diff
@@ -11,17 +14,49 @@ public class SetupHelp
         private const string AddinPartialPackageString = "Quandl";
         private const string AddinPackageString = "Quandl.Excel.UDF.Functions-AddIn.xll";
 
-        private static readonly string OpenValue = $@"/R ""{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}\Quandl\Quandl.Excel.UDF.Functions-AddIn.xll""";
+        private static string Excel32InstallFolder = $@"/R ""C:\Program Files(x86)\Microsoft Office\Office{0}\EXCEL.EXE""";
+    
+        private static string _openValue = null;
+        private static bool _is64Bit = Is64BitOperatingSystem();
+        private static string _version = null;
```
Comment (mbasset) [inline Quandl.Excel.Console/SetupHelp.cs]: I think this needs to be more descriptive. Is this the excel version or the windows version?

File: Quandl.Excel.Console/Version.cs [hunk 3]
```diff
@@ -62,9 +61,9 @@ private bool IsExcel32BitInstall(string version)
                     return Introspection.GetPEFormat(folder) == Introspection.PEFormat.PE32;
                 }
             }
-            catch (Exception e)
+            catch (Exception)
             {
-                Utilities.LogToSentry(e);
```
Comment (mbasset) [inline Quandl.Excel.Console/Version.cs]: So it was logging to sentry that was the error. If we cannot log to sentry can we at least output this error to an installer log file? Maybe in the same directory we install the plugin too?

Comment (junos) [inline Quandl.Excel.Console/Version.cs]: I create the catch all and log function here: https://github.com/quandl/quandl-excel-windows/pull/9/files#diff-d24adddba7dd745b35c4d6213367b5d5R28, the errors for installation will be log into a system temp folder which don't have any write permission issue.

**Pull request merged**